en/3-parachains/1-launch.md

```diff
@@ -58,21 +56,19 @@ parachain-collator \
 ```
 
 The first thing to notice about this command is that several arguments are passed before the lone
-`--`, and several more arguments are passed after it. A cumulus collator contains the actual
-collator node, and also an **embedded relay chain node**. The arguments before the `--` are for the
+`--`, and several more arguments passed after it. A cumulus collator contains the actual
+collator node and also an **embedded relay chain node**. The arguments before the `--` are for the
 collator, and the arguments after the `--` are for the embedded relay chain node.
 
 We give the collator a base path and ports as we did for the relay chain node previously. We also
-specify the parachain id.
+specify the parachain ID.
 
-> Remember to change the collator-specific values if you are executing
-> these instructions a second time for a second parachain.
-> You will use the same relay chain chain spec, but need different ports exposed.
+Remember to change the collator-specific values if you are executing these instructions a second
+time for a second parachain. You will use the same relay chain chain spec, but need different ports
+exposed.
 
-> A Parachain node = (full) collator + (full) validator node.
-> _Eventually_, this will change to only need a minimal light client for the relay chain node.
-> There is also no such thing as a "light" collator node that does not include an embedded
-> relay chain node _yet_ - but there will also eventually be options for this.
+Current, a parachain node is consisted of a full parachain collator with a full relay chain
+validator node.
```

Comment:
```suggestion
> A Parachain node = (full) collator + (full) validator node.
> _Eventually_, this will change to only need a minimal light client for the relay chain node.
> There is also no such thing as a "light" collator node that does not include an embedded
> relay chain node _yet_ - but there will also eventually be options for this.
```